### PR TITLE
Add bparees as mirroring/image-naming approver

### PR DESCRIPTION
**Pull request by bparees**
and add guidelines for image names

### CODE REVIEW

File: core-services/image-mirroring/openshift/GUIDELINES.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+of our images to understand the component/technology/function the image relates to.  This will aid
+them in problem determination and understanding the implications of issues with a particular image.
+
+## Segmentation Ordering
+Image names should follow one the following segmentation orderings:
+
+`$cloudProvider-$component-$optionalModifier` - e.g. aws-ebs-csi-driver-operator
+
+
+`$technology-$component-$optionalModifier` - e.g. sriov-network-config-daemon
```
Comment (smarterclayton) [inline core-services/image-mirroring/openshift/GUIDELINES.md:27]: Maybe add `cluster-*` in a follow up.

File: core-services/image-mirroring/openshift/GUIDELINES.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+to earlier in the "new component" process so contributors don't have to go back and rename a bunch
+of things if their name choice needs to be changed at this step.  Fixing that part of the process
+is TBD.
+
+
+# Guidelines
+
+The purpose of these guidelines are to give consistency to our image names, as well as help consumers
+of our images to understand the component/technology/function the image relates to.  This will aid
+them in problem determination and understanding the implications of issues with a particular image.
```
Comment (smarterclayton) [inline core-services/image-mirroring/openshift/GUIDELINES.md:19]: I would add a sentence.  

> In addition, as image names fall into a common namespace shared by many teams it is important to be able to route and isolate images by both purpose and common, as well as allow groups that work across images (ART, CI, testplatform, security review) to have some common patterns they can rely on.

or similar.

Comment (bparees) [reply core-services/image-mirroring/openshift/GUIDELINES.md:19]: added

File: core-services/image-mirroring/openshift/GUIDELINES.md (hunk 1 above)
Comment (bparees) [reply core-services/image-mirroring/openshift/GUIDELINES.md]: added

File: core-services/image-mirroring/openshift/GUIDELINES.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+For more information on the process of enabling mirroring for an image, see the [CI documentation](https://docs.ci.openshift.org/docs/how-tos/mirroring-to-quay/#mirroring-images).
+
+# Guidelines
+
+The purpose of these guidelines are to give consistency to our image names, as well as help consumers
+of our images to understand the component/technology/function the image relates to.  This will aid
+them in problem determination and understanding the implications of issues with a particular image.
+
+In addition, as image names fall into a common namespace shared by many teams it is important to be able
+to route and isolate images by both purpose and common, as well as allow groups that work across images
```
Comment (dhellmann) [inline core-services/image-mirroring/openshift/GUIDELINES.md:23]: I think there's something missing after the "by both purpose and common" here.

File: core-services/image-mirroring/openshift/GUIDELINES.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+(ART, CI, testplatform, security review) to have some common patterns they can rely on.
+
+## Segmentation Ordering
+Image names should follow one the following segmentation orderings:
+
+`$cloudProvider-$component-$optionalModifier` - e.g. aws-ebs-csi-driver-operator
```
Comment (dhellmann) [inline core-services/image-mirroring/openshift/GUIDELINES.md:29]: There are more dashes in the example than the template, so it's not clear which parts are the component and which are the optional modifier. Can we break out the parts there and say something like, "The image containing X for AWS would be called ..."

File: core-services/image-mirroring/openshift/GUIDELINES.md (hunk 1 above)
Comment (dhellmann) [reply core-services/image-mirroring/openshift/GUIDELINES.md:32]: The same comment about parsing the sample applies here.

**Pull request merged**